libcxx/src/system_error.cpp

```diff
@@ -156,19 +229,52 @@ class _LIBCPP_HIDDEN __system_error_category : public __do_message {
 const char* __system_error_category::name() const noexcept { return "system"; }
 
 string __system_error_category::message(int ev) const {
-#ifdef _LIBCPP_ELAST
+#ifdef _LIBCPP_WIN32API
+  std::string result;
+  char* str               = nullptr;
+  unsigned long num_chars = FormatMessageA(
```

Comment:
```suggestion
  unsigned long num_chars = ::FormatMessageA(
```

That makes it clearer that we're calling a Windows system function.